.travis.yml

```diff
@@ -166,6 +166,7 @@ install:
 before_script:
   - cd "${TRAVIS_BUILD_DIR}"
   - mkdir build && cd build
+  - export CXXFLAGS="$CXXFLAGS -Werror"
```

Comment:
Why not leave this turned on CMakeLists.txt?

Reply:
emacs gave me warnings as errors, and i thought in general its best to just have it on CI